packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.80.0
```

Comment:
the version is v0.80.0, not v0.79.0, so that there's no conflict with this or https://github.com/justeat/fozzie-components/pull/835